.vale/styles/Vocab/General/accept.txt

```diff
@@ -1,8 +1,33 @@
-md
-middleware
-APIs
-API's
-enum
-async
-cfg
-cfg_api
\ No newline at end of file
+md
+middleware
+APIs
+API's
+enum
+async
+cfg
+cfg_api
+pdf
+Pdf
+OAuth
+Oauth
+oauth
+Realtime
+realtime
+postgres
+resends
+onboarding
+onboarded
+ment
+gdpr
+Disenrollment
+Callcenter
+github
+Datadog
+DataDog
+datadog
+Mockhandler
+mockhandler
+Eitri
+eitri
+Unary
+unary
```

Comment:
We should not enable randomish spelling of words, we should stick with the proper ones. Bad spelling reduces the perceived quality of our documentation.

Eg: `github` is spelled `GitHub`, `postgres` is spelled `PostgreSQL`, `oauth` is spelled `OAuth`.

There is not need for duplicate entries to add support for the first letter to be upper case. Instead you write `[Uu]nary`.

There are also words in this list that I could not find in the documentation, these should be removed.

Reply:
@nhhagen I thought you would like to implement a generic vale spell book so Coop can re-use it, but instead, it feels like we will generate N amounts of dictionaries now?

Reply:
We have a separate account effort on that, no need to pre-populate.

Ref:
- https://github.com/coopnorge/github-workflow-techdocs/issues/16